src/lib/swapper/swappers/ThorchainSwapper/utils/addSlippageToMemo.ts

```diff
@@ -29,7 +31,9 @@ export const addSlippageToMemo = ({
     quotedMemo.split(MEMO_PART_DELIMITER)
 
   const limitWithManualSlippage = subtractBasisPointAmount(
-    expectedAmountOutThorBaseUnit,
+    bn(expectedAmountOutThorBaseUnit)
+      .minus(affiliateFeesThorBaseUnit)
+      .toFixed(0, BigNumber.ROUND_DOWN),
```

Comment:
Rounding down or fixing to 0 dp shouldn't really be needed since this is a BN, but just some additional safety and confirming to the `toFixed(0, ROUND_DOWN` below